concepts/05 UI Components/PivotGrid/155 Enhance Performance on Large Datasets.md

```diff
@@ -0,0 +1,23 @@
+This help topic describes how to improve PivotGrid performance when binding and processing large data sets.
+
+[note] To enhance component performance, we recommend you process data before binding it to PivotGrid.
+
+### Enable Remote Operations
+
+The DevExtreme PivotGrid performs all data operations such as filtering and grouping on the client side. To enhance component performance, enable **PivotGridDataSource**.[remoteOperations](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/#remoteOperations) to process data on the server side. PivotGrid **remoteOperations** support [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/) and [XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/) store types. 
```

Comment:
that's not true.

PivotGrid also supports CustomStore (link to API Refs) and createStore (link to https://js.devexpress.com/Angular/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Web_API,_PHP,_MongoDB/). The latter one is quite popular, let's mention this along with a link to a demo: https://js.devexpress.com/Angular/Demos/WidgetsGallery/Demo/PivotGrid/WebAPIService/MaterialBlueLight/

let's remove ODataStore. While it technically works it doesn't support remote grouping and summaries . So PivotGrid will simply load all data from it.